src/derivable/map.ts

```diff
@@ -101,6 +101,7 @@ export const isUnresolvedOrErrorWrapper = (state: State<any>): state is typeof u
     state === unresolved || state instanceof ErrorWrapper;
 
 export function mapMethod<B, V>(this: BaseDerivable<B>, get: (b: B) => MaybeFinalState<V>, set?: (v: V, b?: B) => B): Derivable<V> {
+    // tslint:disable-next-line:no-shadowed-variable
```

Comment:
Can this be solved without the lint ignore by renaming either of the `this` parameters?

Reply:
disabled this tslint rule at tslint.json